bpf/go_common.h

```diff
@@ -401,4 +401,37 @@ static __always_inline void *unwrap_tls_conn_info(void *conn_ptr, void *tls_stat
     return conn_ptr;
 }
 
+static __always_inline void process_meta_frame_headers(void *frame, tp_info_t *tp) {
+    if (frame) {
+        void *fields = 0;
+        bpf_probe_read(&fields, sizeof(fields), (void *)(frame + 8));
+        u64 fields_len = 0;
+        bpf_probe_read(&fields_len, sizeof(fields_len), (void *)(frame + 8 + 8));
+        bpf_dbg_printk("fields ptr %llx, len %d", fields, fields_len);
+        if (fields && fields_len > 0) {
```

Comment:
idem

Reply:
Maybe my preference is otherwise :), indenting is a problem, but returns in middle of text are easy to miss.